lib/spack/spack/cmd/create.py

```diff
@@ -283,10 +293,8 @@ class RacketPackageTemplate(PackageTemplate):
     """Provides approriate overrides for Racket extensions"""
     base_class_name = 'RacketPackage'
 
-    url_line = """\
-    # FIXME: set the proper location from which to fetch your package
-    git      = "git@github.com:example/example.git"
-    """
+    url_line = "    # FIXME: set the proper location from which to fetch your package"
+    git_line = "    git      = 'git@github.com:example/example.git'"
```

Comment:
More than 99% of the packages that define the `git` property use the `https` protocol so this statement would be:

```suggestion
    git_line = "    git      = 'https://github.com/example/example.git'"
```

Of those that don't, the protocol is`ssh:` or

```suggestion
    git_line = "    git      = 'ssh://git@github.com/example/example.git'"
```